store/storetest/channel_store.go

```diff
@@ -576,12 +576,12 @@ func testChannelStoreGetByName(t *testing.T, ss store.Store) {
 		t.Fatal("Missing id should have failed")
```

Comment:
Some lines before (not included in this change) there is an `}else{` after a `if .... t.Fatal`, the idiomatic way to solve this, is the used in this change. so, for consistency, i would use the same in both places.

Reply:
I rewrote most of the test. Please take a fresh look.